examples/latte/README.md

```diff
@@ -84,12 +92,17 @@ Some of the generated results are shown here:
     <td width=33% style="border: none"><img src="https://raw.githubusercontent.com/wtomin/mindone-assets/main/latte/sky/generated-2.gif" style="width:100%"></td>
     </tr>
 </table>
+<p align="center">
+  <em> Figure 2. The generated videos of the pretrained model converted from the torch checkpoint. </em>
+</p>
+
+## 4. Training
 
-## Training
+### 4.1 Training On-the-Fly
```

Comment:
`On-the-fly` is somehow ambiguous in these scenario, maybe choose another word

Reply:
Agreed. Perhaps, `Training with Videos`?

Reply:
Agreed. Perhaps, `Training with Videos`?

Reply:
Great advice, I have corrected it.